_sass/blog.scss

```
@@ -195,3 +195,68 @@ figcaption {
+        box-shadow: 0 2px 10px var(--brand-color-secondary-50);
+        text-decoration: none;
+      }
+    }
+  }
+
+  .category-card-name {
+    color: var(--color-secondary-dark);
+    font-size: 1.1rem;
+    font-weight: 500;
+    text-transform: capitalize;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)